swan-lake/learn-the-platform/ballerina-persist/overview.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: ballerina-persist-documentation-left-nav-pages-swanlake
+title: Bal persist overview
+description: The section gives an overview of the `bal persist` feature.
+keywords: ballerina, programming language, ballerina packages, persist, persist model, persist cli, persist client api
+permalink: /learn/ballerina-persist/persist-overview/
+active: persist_overview
+intro: The `bal persist` feature allows you to store data in different data stores and retrieve them when needed. A data store can be a database, an in-memory cache, or a file system. The bal persist feature currently supports In-memory table, MySQL database, and Google Sheets as data stores. The important point is you can use the same syntax to access data in all these data stores. So you don't need to learn different syntaxes to access data in different data stores.
+redirect_from:
+- /learn/ballerina-persist/persist-overview/
+---
+This feature has three main sections: Persist Model, Persist CLI and Persist Client API. The Persist Model is used to define the data model. The Persist CLI is used to generate the client API for the data model. The Persist Client API is used to access the data in the data store.
```

Comment:
```suggestion
This feature has three main components: Persist Model, Persist CLI, and Persist Client API. The Persist Model is used to define the data model. The Persist CLI is used to generate the client API for the data model. The Persist Client API is used to access the data in the data store.
```
Shall we add the 3 components as a list along with the descriptions? These descriptions might be redundant as we have sections below with all 3 described in detail.